cluster/addons/addon-manager/Makefile

```diff
@@ -15,10 +15,10 @@
 IMAGE=gcr.io/k8s-staging-addon-manager/kube-addon-manager
 ARCH?=amd64
 TEMP_DIR:=$(shell mktemp -d)
-VERSION=v9.1.7
-KUBECTL_VERSION?=v1.27.1
+VERSION=v9.1.8
+KUBECTL_VERSION?=v1.28.2
 
-BASEIMAGE=registry.k8s.io/debian-base-$(ARCH):v1.0.1
+BASEIMAGE=registry.k8s.io/debian-base-$(ARCH):v2.0.0
```

Comment:
https://github.com/kubernetes/kubernetes/blob/09200e9c92784f6fb9210e6287e306649a731673/cluster/images/etcd/Makefile#L95-L96 is using the `bookworm-v1.0.0`. 

https://github.com/kubernetes/kubernetes/blob/09200e9c92784f6fb9210e6287e306649a731673/build/dependencies.yaml#L145-L177
This needs an update if we want to migrate to v2.0.0.

Reply:
@pacoxu thanks! i updated to use `debian-base:bookworm-v1.0.0` if that works